[stable/filebeat] filebeat fixes

rtluckie opened the pull request:
- remove opinionated tolerations
- use oss image
- bump image version to 6.3.0
- include `/var/log/messages` and `/var/log/syslog`
- fix indent in .Values.config
- bump chart version
- bump appVersion

Commit:
Filebeat fixes

- remove opinionated tolerations
- use oss image
- bump image version to 6.3.0
- include `/var/log/messages` and `/var/log/syslog`
- fix indent in .Values.config
- bump chart version
- bump appVersion

sstarcher commented on stable/filebeat/values.yaml:
```diff
@@ -75,6 +77,12 @@ resources: {}
 
+nodeSelector: {}
+
+tolerations: []
```
please put the "opinionated" toleration here as users will expect this to collect all logs.

rtluckie replied:
@sstarcher fixed

rendhalver replied:
What about the other tolerations?
```
      - key: node-role.kubernetes.io/master
        operator: Exists
        effect: NoSchedule
```
You have missed all the elements of the original toleration

dcwangmit01 replied:
@rendhalver The toleration that is in this PR, but commented out by default and serving as an example:

```
tolerations: []
#  - operator: Exists
```
Should match everything.

From the [docs](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration):
```
    An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything.

tolerations:
- operator: "Exists"
```

rendhalver replied:
I don't think you are correct.
The - indicates it's an array and the whole hash i pasted is one element of that array.
You have only included one part of that sub hash.

rendhalver replied:
Oh I see what you are doing.
What does removing the rest of elements of the hash achieve?
Does it do the same thing as the original?
The main reason I am asking is people have probably already used this to deploy filebeat and we could be changing how it functions.

Commit:
requested changes

rendhalver commented on stable/filebeat/Chart.yaml:
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
-version: 0.3.0
-appVersion: 6.2.3
+version: 0.3.1
+appVersion: 6.3.0
```
Is increasing the image version really a bugfix?
Increasing the minor version of the appVersion and image seems like cause for increasing at least the minor version of the chart.

rtluckie replied:
@rendhalver  good catch!
fixed.

Commit:
requested changes

- bump chart minor version
- bump filebeat image version to 6.3.1

Commit:
Requested changes

- update default `.Values.tolerations` to ensure filebeat pod is scheduled on every node (including masters)
